lib/ain-grpc/src/rpc/web3.rs

```diff
@@ -19,8 +19,9 @@ pub struct MetachainWeb3RPCModule {}
 
 impl MetachainWeb3RPCServer for MetachainWeb3RPCModule {
     fn client_version(&self) -> RpcResult<String> {
-        let version = env!("CARGO_PKG_VERSION");
-        let commit = option_env!("GIT_HASH").unwrap_or("unknown");
+        let version: String = ain_cpp_imports::get_client_version();
+        let commit = option_env!("GIT_HASH")
```

Comment:
We can remove the need to get the hash from the local env during runtime, when we can directly get it from the compiled binary through the cpp side and expose it on the FFI as well.

Reply:
agree, good point 

lemme make a new PR for this

Reply:
No worries I'm making the fixes directly, all good.

Reply:
any good first issues that i can look into :) 

@sieniven

Reply:
Resolved directly on #2107.